javacutil/src/main/java/org/checkerframework/javacutil/TypesUtils.java

```diff
@@ -384,8 +384,7 @@ public static boolean isBoxedPrimitive(TypeMirror type) {
     public static boolean isImmutableTypeInJdk(TypeMirror type) {
         return isPrimitive(type)
```

Comment:
I looked at how `isPrimitive` is implemented... Do you understand why that isn't just `type.getKind().isPrimitive()`?

Reply:
I think it should just be `type.getKind().isPrimitive()`... I noticed other duplicate logic and also removed them.